docs/commands/command-line-interface.md

```diff
@@ -36,8 +36,8 @@ _Valid for both nano_node and nano_wallet processes_
 Pass node configuration values. This takes precedence over any values in the configuration file. This option can be repeated multiple times.
 
 ### --confirmation_height_clear
-_version 19.0+_  
```

Comment:
I think we want to keep both. We tend to describe the API changes going back historically.

Reply:
For example:
after v24:
* this changed
* that changed
after v19:
* this changed
* that changed